mlfinlab/tests/test_sb_bagging.py

```diff
@@ -0,0 +1,310 @@
+                                                                  triple_barrier_events=self.meta_labeled_events,
+                                                                  price_bars=self.data, max_features='20')
+        bagging_clf_4 = SequentiallyBootstrappedBaggingClassifier(base_estimator=clf,
+                                                                  triple_barrier_events=self.meta_labeled_events,
+                                                                  price_bars=self.data, max_features=2000000)
+        bagging_clf_5 = SequentiallyBootstrappedBaggingClassifier(base_estimator=clf,
+                                                                  triple_barrier_events=self.meta_labeled_events,
+                                                                  price_bars=self.data, oob_score=True, warm_start=True)
+        bagging_clf_6 = SequentiallyBootstrappedBaggingClassifier(base_estimator=clf,
+                                                                  triple_barrier_events=self.meta_labeled_events,
+                                                                  price_bars=self.data, warm_start=True)
+        bagging_clf_7 = SequentiallyBootstrappedBaggingClassifier(base_estimator=clf,
+                                                                  triple_barrier_events=self.meta_labeled_events,
+                                                                  price_bars=self.data, warm_start=True)
+        with self.assertRaises(ValueError):
+            bagging_clf_1.fit(self.X_train, self.y_train_clf, sample_weight=np.ones((self.X_train.shape[0],)), )
+        with self.assertRaises(ValueError):
+            bagging_clf_2.fit(self.X_train, self.y_train_clf,
+                              sample_weight=np.ones((self.X_train.shape[0],)), )
+        with self.assertRaises(ValueError):
+            bagging_clf_3.fit(self.X_train, self.y_train_clf,
+                              sample_weight=np.ones((self.X_train.shape[0],)), )
+        with self.assertRaises(ValueError):
+            bagging_clf_4.fit(self.X_train, self.y_train_clf,
+                              sample_weight=np.ones((self.X_train.shape[0],)), )
+        with self.assertRaises(ValueError):
+            bagging_clf_5.fit(self.X_train, self.y_train_clf,
+                              sample_weight=np.ones((self.X_train.shape[0],)), )
+        with self.assertRaises(ValueError):
+            bagging_clf_6.fit(self.X_train, self.y_train_clf)
+            bagging_clf_6.n_estimators -= 2
+            bagging_clf_6.fit(self.X_train, self.y_train_clf)
+
+        with self.assertRaises(ValueError):
+            bagging_clf_7.fit(self.X_train, self.y_train_clf)
+            bagging_clf_7.n_estimators -= 1000
+            bagging_clf_7.fit(self.X_train, self.y_train_clf)
+
+    def test_sb_classifier(self):
+        """
+        Test Sequentially Bootstrapped Bagging Classifier
```

Comment:
Better doc string, please. What specific thing is it testing? At first glance, it looks like you are comparing a sklearn ensemble with the SB ensemble?